docs/api/info/get-fee-rate.schema.example.json

```diff
@@ -0,0 +1,3 @@
+{
+  "fee_rate": 360
```

Comment:
Are the units obvious to everyone? I feel like somewhere in the OpenAPI docs or comments we should cover how exactly to use fee_rate to derive the fee. Thoughts @kyranjamie @zone117x ?

Reply:
It's not really clear to me. IIUC, the unit currently used by `/v2/fees/transfer` is simply byte length, which was a simple heuristic shipped with Stacks 2.0 but not very accurate? It seems like in the future (e.g. Stacks 2.1) there would be something like a fee _rate_ for multiple types of execution limits (read, write, runtime, etc), as opposed to a single value like this.

Given how fee estimation currently works, a simple fee multiplier makes more sense to me. This is what was just implemented in the send-many CLI.

Reply:
Could use some guidance from @reedrosenbluth @kantai